#! /bin/sh /usr/share/dpatch/dpatch-run
## 49-missing-template.dpatch by Thomas Girard <thomas.g.girard@free.fr>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Unbreak auto* macros. Thanks to Julien Lavergne for the patch.

@DPATCH@
--- ACE_wrappers/m4/ace_defines.m4	2008/09/18 18:56:13	82783
+++ ACE_wrappers/m4/ace_defines.m4	2008/09/18 21:22:05	82784
@@ -20,12 +20,12 @@
 #-----------------------------------------------------------------------------
 AC_DEFUN([ACE_CHECK_HAS_DEFINES],
 [AC_FOREACH([ACE_Def], [$1],
-  [AH_TEMPLATE(AS_TR_CPP(ACE_HAS_[]ACE_Def),
+  [AH_TEMPLATE(AS_TR_CPP([ACE_HAS_]ACE_Def),
                [Define to 1 if platform has ]ACE_Def[().])])dnl
 for ace_def in $1
 do
 ACE_CHECK_DEFINE($ace_def,
-                 [AC_DEFINE_UNQUOTED([AS_TR_CPP([ACE_HAS_$ace_def])]) $2],
+                 [AC_DEFINE_UNQUOTED(AS_TR_CPP([ACE_HAS_$ace_def])) $2],
                  [$3],
                  [$4])dnl
 done
@@ -36,13 +36,13 @@
 #-----------------------------------------------------------------------------
 AC_DEFUN([ACE_CHECK_LACKS_DEFINES],
 [AC_FOREACH([ACE_Def], [$1],
-  [AH_TEMPLATE(AS_TR_CPP(ACE_LACKS_[]ACE_Def),
+  [AH_TEMPLATE(AS_TR_CPP([ACE_LACKS_]ACE_Def),
                [Define to 1 if platform lacks ]ACE_Def[().])])dnl
 for ace_def in $1
 do
 ACE_CHECK_DEFINE($ace_def,
                  [$2],
-                 [AC_DEFINE_UNQUOTED([AS_TR_CPP([ACE_LACKS_$ace_def])]) $3],
+                 [AC_DEFINE_UNQUOTED(AS_TR_CPP([ACE_LACKS_$ace_def])) $3],
                  [$4])dnl
 done
 ])
--- ACE_wrappers/m4/ace_functions.m4	2008/08/13 14:17:28	82621
+++ ACE_wrappers/m4/ace_functions.m4	2008/09/18 21:22:05	82784
@@ -2,12 +2,12 @@
 #---------------------------------------------------------------------------
 AC_DEFUN([ACE_CHECK_HAS_FUNCS],
 [AC_FOREACH([ACE_Func], [$1],
-  [AH_TEMPLATE(AS_TR_CPP(ACE_HAS_[]ACE_Func),
+  [AH_TEMPLATE(AS_TR_CPP([ACE_HAS_]ACE_Func),
                [Define to 1 if platform has ]ACE_Func[().])])dnl
 for ace_func in $1
 do
 AC_CHECK_FUNC($ace_func,
-              [AC_DEFINE_UNQUOTED([AS_TR_CPP([ACE_HAS_$ace_func])]) $2],
+              [AC_DEFINE_UNQUOTED(AS_TR_CPP([ACE_HAS_$ace_func])) $2],
               [$3])dnl
 done
 ])
@@ -16,12 +16,12 @@
 #-----------------------------------------------------------------------------
 AC_DEFUN([ACE_CHECK_LACKS_FUNCS],
 [AC_FOREACH([ACE_Func], [$1],
-  [AH_TEMPLATE(AS_TR_CPP(ACE_LACKS_[]ACE_Func),
+  [AH_TEMPLATE(AS_TR_CPP([ACE_LACKS_]ACE_Func),
                [Define to 1 if platform lacks ]ACE_Func[().])])dnl
 for ace_func in $1
 do
 AC_CHECK_FUNC($ace_func,
               [$2],
-              [AC_DEFINE_UNQUOTED([AS_TR_CPP([ACE_LACKS_$ace_func])]) $3])dnl
+              [AC_DEFINE_UNQUOTED(AS_TR_CPP([ACE_LACKS_$ace_func])) $3])dnl
 done
 ])
--- ACE_wrappers/m4/ace_headers.m4	2008/08/13 14:17:28	82621
+++ ACE_wrappers/m4/ace_headers.m4	2008/09/18 21:22:05	82784
@@ -5,12 +5,12 @@
 #-----------------------------------------------------------------------------
 AC_DEFUN([ACE_CHECK_HAS_HEADERS],
 [AC_FOREACH([ACE_Header], [$1],
-  [AH_TEMPLATE(AS_TR_CPP(ACE_HAS_[]ACE_Header),
+  [AH_TEMPLATE(AS_TR_CPP([ACE_HAS_]ACE_Header),
                [Define to 1 if platform has the <]ACE_Header[> header file.])])dnl
 for ace_header in $1
 do
 AC_CHECK_HEADER($ace_header,
-                [AC_DEFINE_UNQUOTED([AS_TR_CPP([ACE_HAS_$ace_header])]) $2],
+                [AC_DEFINE_UNQUOTED(AS_TR_CPP([ACE_HAS_$ace_header])) $2],
                 [$3],
 	        [$4])dnl
 done
@@ -23,13 +23,13 @@
 #-----------------------------------------------------------------------------
 AC_DEFUN([ACE_CHECK_LACKS_HEADERS],
 [AC_FOREACH([ACE_Header], [$1],
-  [AH_TEMPLATE(AS_TR_CPP(ACE_LACKS_[]ACE_Header),
+  [AH_TEMPLATE(AS_TR_CPP([ACE_LACKS_]ACE_Header),
                [Define to 1 if platform lacks the <]ACE_Header[> header file.])])dnl
 for ace_header in $1
 do
 AC_CHECK_HEADER($ace_header,
                 [$2],
-                [AC_DEFINE_UNQUOTED([AS_TR_CPP([ACE_LACKS_$ace_header])]) $3],
+                [AC_DEFINE_UNQUOTED(AS_TR_CPP([ACE_LACKS_$ace_header])) $3],
                 [$4])dnl
 done
 ])
